mycroft/engines/intent_engine.py

```diff
@@ -49,17 +49,19 @@ def __init__(self, path_manager):
     def try_register_intent(*args, **kwargs):
         """
         Attempt to register intent with given arguments
-        :rtype str
-        :returns intent name if parsed parameters, otherwise ""
+        Returns:
+            name (str): intent name if parsed parameters, otherwise ""
         """
         pass
 
     @abstractmethod
     def calc_intents(self, query):
         """
         Run the intent engine to determine the probability of each intent against the query
-        :param query: input sentence as a single string
-        :return: dict of intent: intent_data where
+        Args:
+            query (str): input sentence as a single string
+        Returns:
+            intent (dict): intent_data where
```

Comment:
Ok, I was wondering about that one and then skipped it until I could figure out what it was doing.
I can try something.